lib/ancestry/instance_methods.rb

```diff
@@ -17,13 +17,7 @@ def update_descendants_with_new_ancestry
           # ... replace old ancestry with new ancestry
           descendant.without_ancestry_callbacks do
             descendant.update_attribute(
-              self.ancestry_base_class.ancestry_column,
-              descendant.read_attribute(descendant.class.ancestry_column).gsub(
-                # child_ancestry_was
-                /^#{self.child_ancestry}/,
-                # future child_ancestry
-                if ancestors? then "#{read_attribute self.class.ancestry_column }/#{id}" else id.to_s end
-              )
```

Comment:
I am still trying to understand what this code did, but I can see why this might have been your rationale for wanted to make this switch a bit more now 😅

Reply:
thanks for this comment - think I introduced `ancestry_ids_was` - when I should be going towards the whole `ancestor_ids_before_last_save` route

Reply:
Well, I don't know that comment was worth thanking me for, I just understand the rational a bit better now that I have had the time to understand it a bit better.